src/components/institution_table/InstitutionDetails.tsx

```diff
@@ -64,6 +65,41 @@ export const InstitutionDetails = (props: InstitutionDetailsProps) => {
         loadInstitution();
     }, [formMode, institutionId]);
 
+    const normalizeInstitutionName = (name: string): string => {
+        // Trim whitespace from start and end
+        let normalized = name.trim();
+
+        // Replace curly single quotes with straight straight Iquotes
+        normalized = normalized.replace(/[‘’]/g, "'");
```

Comment:
I'm wondering if we need to communicate to the user that we've changed their input.